nginx/config.go

```diff
@@ -201,7 +201,7 @@ http {
 			proxy_set_header Upgrade $http_upgrade;
 			proxy_set_header Connection $connection_upgrade;
 
-			{{ if $enforceHTTPS }}if ($access_scheme != "https") {
+			{{ if or $enforceHTTPS $appConfig.SSL.Enforce }}if ($access_scheme != "https") {
 				return 301 https://$host$request_uri;
```

Comment:
Shouldn't the redirect keep the query string? That can be done like shown below.

```
return 301 https://$host$request_uri$is_args$args;
```